jsignpdf/src/main/resources/net/sf/jsignpdf/translations/messages.properties

```diff
@@ -13,7 +13,7 @@ console.createOutPdf=Creating output PDF file: {0}
 console.createPdfReader=Opening input PDF file: {0}
 console.createSignature=Creating signature
 console.creatingTsaClient=Creating TSA client.
-console.criticalExtensionNotSupported=Certificate {0} contains critical extension with OID "{1}" which is not recognized by JSignPdf - skipping.
+console.criticalExtensionNotSupported=The {0} certificate contains a critical (OID "{1}") extension not recognized by JSignPDF — skipping.
```

Comment:
```suggestion
console.criticalExtensionNotSupported=The {0} certificate contains a critical extension (OID "{1}") not recognized by JSignPdf - skipping.
```

* We should leave the name to be `JSignPdf`, it's used in the majority of places (web, jar-name, ...)
* Also I would keep the `console.*` and `hlp.*` messages ASCII-based as some terminals (pointing on you, Windows `cmd`) could have issues with "advanced" character (`—`, `…`)